README.md

```diff
@@ -323,7 +323,7 @@ render() {
       </Col>
     </div>
   );
-}
+}}
```

Comment:
I think this would actually break App.js — the code block here refers only to `render()`, the class should already be closing.